training/platform-services/PLATFORM.md

```diff
@@ -0,0 +1,182 @@
+
+More information about Vault can be found on their website - https://www.vaultproject.io/
+
+## Grafana
+Visit the following to access Grafana
+- DEVELOP:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;https://grafana.develop.onsdigital.co.uk
+- PRODUCTION:&nbsp;&nbsp;&nbsp;https://grafana.production.onsdigital.co.uk
+
+[<img src="images/grafana-dev-home.png" title="Grafana Develop Home" width=49.7% height=49.7%/>][grafana-dev-home] [<img src="images/grafana-dev-nomad-task.png" title="Grafana Develop Nomad Task" width=49.7% height=49.7%/>][grafana-dev-nomad-tasks]
+
+Grafana informs how a process is behaving on the machine (its use of CPU, memory, use of IO operations, etc). In Grafana, the [Nomad Task][grafana-dev-nomad-tasks] dashboard is commonly used, which gives a list of tasks (each related to a specific application). From there, you can see the behaviours of specific applications to the machine such as memory usage (e.g. to identify a potential memory leak by changing the range of time to 7 days and analysing the graph). Grafana assists in debugging and we can look at [Kibana](#kibana) to see logs at a particular time based on any anomalies discovered from graphs in Grafana. Furthermore, the [Node Stats 2][grafana-dev-node-stats-2] dashboard is used as well which is specific to EC2 instances performance.
+
+Please note that CPU can spike as reserved CPU can exceed the limit because of its soft limit (CPU bursting). On the other hand, memory usage cannot spike as a hard limit is set for reserved memory and if memory exceeds its limit, the service will stop and restart
+
+More information about Grafana can be found on their website - https://grafana.com/docs/grafana/latest/
+
+## Prometheus
+Visit the following to access Prometheus
+- DEVELOP:&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;https://prometheus.develop.onsdigital.co.uk
+- PRODUCTION:&nbsp;&nbsp;&nbsp;https://prometheus.production.onsdigital.co.uk
+
+[<img src="images/prometheus-dev-home.png" title="Prometheus Develop Home" width=49.7% height=49.7%/>][prometheus-dev-home] [<img src="images/prometheus-dev-alerts.png" title="Prometheus Develop Alerts" width=49.7% height=49.7%/>][prometheus-dev-alerts]
+
+Prometheus provides alerts and alarms to notify us on technical issues. These alerts set in Prometheus is alarmed in Slack. Prometheus stores all data as time series in which a time series graph can be viewed to report information. Therefore, more details about the alert can be retrieved from Prometheus. Prometheus uses the flexible query language [PromQL][prometheus-querying] to let us select and aggregate time series data in real time, speeding up the process of debugging
```

Comment:
> Prometheus stores all data as time series in which a time series graph can be viewed to report information. Therefore, more details about the alert can be retrieved from Prometheus. 

I feel like this isn't really telling me a lot, but can't put my finger on what to say instead. Maybe something like 'Prometheus stores all data as a time series, and monitors for changes over time. The information can be viewed in prometheus in a chart, and these charts are linked to from alerts so more detail is available as you investigate.' Does that make sense and still convey the same things? Sometimes I find in trying to define things, we use the same word too much and it gets hard to follow a bit (but that might just be me!)

Reply:
Yeah. That makes sense
Agreed and done